_std/defines/speech_defines/chat_prefixes.dm

```diff
@@ -0,0 +1,18 @@
+#define CHAT_PREFIX_RADIO ":"
+#define CHAT_PREFIX_RADIO_GENERAL ";"
+
+#define CHAT_PREFIX_AI_RADIO_1 ":1"
+#define CHAT_PREFIX_AI_RADIO_2 ":2"
+#define CHAT_PREFIX_AI_RADIO_3 ":3"
+
+#define CHAT_PREFIX_DEADCHAT ":d"
+#define CHAT_PREFIX_HIVECHAT ":hive"
+#define CHAT_PREFIX_INTERCOM ":in"
+#define CHAT_PREFIX_KUDZUCHAT ":kudzu"
+#define CHAT_PREFIX_THRALLCHAT ":thrall"
+#define CHAT_PREFIX_SILICONCHAT ":s"
+#define CHAT_PREFIX_LEFTHANDCHAT ":lh"
+#define CHAT_PREFIX_RIGHTHANDCHAT ":rh"
```

Comment:
These should all follow the common naming scheme set out by the `SPEECH_PREFIX_` defines. I.e. `SILICON`, `LEFT_HAND`, and `RIGHT_HAND`.